ntbs-service/Startup.cs

```diff
@@ -106,6 +107,18 @@ public void ConfigureServices(IServiceCollection services)
 
                         return Task.CompletedTask;
                     };
+
+                    options.Events.OnSecurityTokenValidated += context =>
+                    {
+                        var username = context.Principal.Claims
```

Comment:
This could just be:
`context.Principal.FindFirstValue(ClaimTypes.Email);`

Which would match with other places we're doing this functionality